scanpy/tests/test_metrics.py

```diff
@@ -0,0 +1,97 @@
+from operator import eq
+from string import ascii_letters
+
+import numpy as np
+import pandas as pd
+import scanpy as sc
+from scipy import sparse
+
+
+def test_gearys_c():
```

Comment:
would be great if you could split this up into multiple tests (or parametrize the one)

Reply:
I've split it up based on data, but many of tests are checking that calling the method different ways is equivalent.